src/model_io/urdf/include/iDynTree/ModelIO/ModelExporter.h

```diff
@@ -42,6 +42,19 @@ class ModelExporterOptions
      */
     std::string baseLink;
 
+    /**
+     * Select if the first additional frame of the base link is exported as fake base link.
+     *
+     * The URDF exporter by default exports the first additional frame of the base link as
+     * a parent "fake" link to the actual base link, as a workaround for https://github.com/ros/kdl_parser/issues/27).
+     * By setting this option to false, is possible to disable this behaviour, for more info see iDynTree::ModelExporter docs.
+     * This option is ignored in non-URDF exporter.
+     *
+     * Default value: true.
+     * Supported formats: urdf.
+     */
+    bool exportFirstBaseLinkAdditionalFrameAsFakeURDFBase{true};
+
```

Comment:
I do not quite understand what might happen if the base link has more than one additional frames and you do not want the first additional link as the fake base link?

I see here https://github.com/traversaro/idyntree/blob/b780bd770a2a2960833cea8abf159a3e576f73c0/src/model_io/urdf/src/URDFModelExport.cpp#L350
we are handling only one additional frame scenario.

Maybe we should create an issue for this to be handled later if it's not necessary now.

And by this feature, we handle additionally handle a case if we do not want the first additional frame to be default fake-base.

Reply:
> I do not quite understand what might happen if the base link has more than one additional frames and you do not want the first additional link as the fake base link?

In that case, `baseFakeLinkFrameIndex` remains set to `FRAME_INVALID_INDEX`, and all the additional frames of the base link get exported as child fake links in https://github.com/traversaro/idyntree/blob/b780bd770a2a2960833cea8abf159a3e576f73c0/src/model_io/urdf/src/URDFModelExport.cpp#L394 .

Reply:
I see. 

> In that case, `baseFakeLinkFrameIndex` remains set to `FRAME_INVALID_INDEX`, and all the additional frames of the base link get exported as child fake links 

Do you think this goes in the documentation?

Reply:
I approved the PR anyways because it was just my personal doubt. Maybe I was a bit confused with the Model and ModelExporter differences ;D

Reply:
> I see.
> 
> > In that case, `baseFakeLinkFrameIndex` remains set to `FRAME_INVALID_INDEX`, and all the additional frames of the base link get exported as child fake links
> 
> Do you think this goes in the documentation?

Added in https://github.com/robotology/idyntree/pull/567/commits/1db7d57de53aee9fec797ce13c682b6c287029d1 .